### ISSUE TITLE
i18n: translate cookie consent buttons

### ISSUE THREAD

**Issue by Guts** [squidfunk/mkdocs-material#3205]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Cookie consent buttons are not translated.

### Expected behaviour

Add buttons translations 

### Actual behaviour

As [stipulated here](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#cookie-consent), title and text can be translated. But not the buttons.

### Steps to reproduce

1. set site language to something else than English
2. open cookie consent

### Package versions

- Python: `python --version`
- MkDocs: `mkdocs --version`
- Material: `pip show mkdocs-material | grep -E ^Version`

### Configuration

```yaml
theme:
  name: "material"

  language: fr

extra:
  analytics:
    provider: google
    property: !ENV MKDOCS_GOOGLE_ANALYTICS_KEY
  consent:
    title: Consentement 🍪
    description: >
      Nous utilisons des cookies pour le goûter mais aussi pour vous proposer
      des fonctionnalités (commentaires, thème clair ou sombre, etc.) et mesurer
      l'audience du site.
```

### System information

ND

**Comment by squidfunk** [comment]
Thanks for suggesting. We can two new values as translations. What are the French equivalents of "Manage settings" and "Accept"?

**Comment by Guts** [comment]
Manage settings: Paramétrer vos choix
Accept: Accepter

**Comment by squidfunk** [comment]
[It's in fact already translated](https://github.com/squidfunk/mkdocs-material-insiders/blob/d0e603cc5e023c4c342537ed31e0e12258032808/src/partials/consent.html#L52-L59), I forgot. The French translations were just missing, which is why it defaulted to English. I've added them in [`5d9887ea3`](https://github.com/squidfunk/mkdocs-material-insiders/commit/5d9887ea305c85e06fc6edac90f9961e70e20a25). I'll clean up and few things and issue a new release shortly.

**Comment by squidfunk** [comment]
This was just released as part of 7.3.6-insiders-3.2.3.
